circuit_maintenance_parser/parsers/lumen.py

```diff
@@ -0,0 +1,100 @@
+"""Lumen parser."""
+import logging
+from typing import Dict
+
+import dateutil.parser as parser
+import bs4  # type: ignore
+from bs4.element import ResultSet  # type: ignore
+
+from circuit_maintenance_parser.errors import ParsingError
+from circuit_maintenance_parser.parser import Html, Impact, CircuitImpact, Status
+
+# pylint: disable=too-many-nested-blocks,no-member, too-many-branches
+
+
+logger = logging.getLogger(__name__)
+
+
+class ParserLumen(Html):
+    """Notifications Parser for Lumen notifications."""
+
+    provider_type: str = "lumen"
+
+    # Default values for Lumen notifications
+    _default_provider = "lumen"
+    _default_organizer = "smc@lumen.com"
+
+    def parse_html(self, soup, data):
+        """Execute parsing."""
+        try:
+            self.parse_spans(soup.find_all("span"), data)
+            self.parse_tables(soup.find_all("table"), data)
+
+        except Exception as exc:
+            raise ParsingError from exc
+
+    def parse_spans(self, spans: ResultSet, data: Dict):
+        """Parse Span tag."""
+        for line in spans:
+            if isinstance(line, bs4.element.Tag):
+                if line.text.lower().strip().startswith("scheduled maintenance window #:"):
+                    data["maintenance_id"] = line.text.lower().strip().split("#: ")[-1]
+                elif line.text.lower().strip().startswith("summary:"):
+                    for sibling in line.next_siblings:
+                        text = sibling.text if isinstance(sibling, bs4.element.Tag) else sibling
```

Comment:
Similarly, save yourself from needing to call `strip()` over and over again:

```suggestion
                        text = sibling.text.strip() if isinstance(sibling, bs4.element.Tag) else sibling.strip()
```